lib/fingerprint/command/scan.rb

```diff
@@ -37,7 +37,7 @@ class Scan < Samovar::Command
 				option "-p/--path <path>", "Analyze the given path relative to root.", default: "./"
 				
 				option "-x/--extended", "Include extended information about files and directories."
-				option "-s/--checksums <MD5,SHA1>", "Specify what checksum algorithms to use: #{CHECKSUMS.keys.join(', ')}.", default: DEFAULT_CHECKSUMS, type: Checksums
+				option "-s/--checksums <SHA2.256>", "Specify what checksum algorithms to use: #{CHECKSUMS.keys.join(', ')}.", default: DEFAULT_CHECKSUMS, type: Checksums
```

Comment:
Doesn't that first arg in the angle brackets show what the defaults are? If you leave two entries in there with commas you are not really reflecting the default.  Not sure why defaulting to always using two different hashes would be beneficial.